src/Native/System.Security.Cryptography.Native/evp.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Use the name pal_evp.cpp to match the pattern we've got elsewhere (except for the grandfathered openssl.c)